mikrotik/resource_generic_crud_operations.go

```diff
@@ -61,7 +61,11 @@ func GenericReadResource(terraformModel interface{}, mikrotikModel client.Resour
 			return
 		}
 
-		resource, err := client.Find(mikrotikModel)
+		resource, err := mikrotikClient.Find(mikrotikModel)
+		if client.IsNotFoundError(err) {
```

Comment:
this check was missing, so it was a bug.
I didn't create a separate pair of issue/PR for it, but let me know if you want it to be split for better visibility.

Reply:
I think it's ok to fix it inline here.